package.json

```diff
@@ -1,47 +1,50 @@
 {
-  "name": "@lokalise/your-project-name",
-  "version": "1.0.0",
-  "scripts": {
-    "build": "vite build",
-    "lint:eslint": "eslint --cache . --ext .js,.cjs,.ts",
-    "lint:ts": "tsc --noEmit",
-    "lint": "npm run lint:eslint && npm run lint:ts",
-    "lint:fix": "npm run lint:eslint -- --fix",
-    "test": "vitest run",
-    "prepare": "husky install",
-    "commit": "commit"
-  },
-  "type": "module",
-  "files": [
-    "dist"
-  ],
-  "main": "./dist/your-project-name.umd.cjs",
-  "module": "./dist/your-project-name.js",
-  "exports": {
-    ".": {
-      "import": "./dist/your-project-name.js",
-      "require": "./dist/your-project-name.umd.cjs"
-    }
-  },
-  "devDependencies": {
-    "@commitlint/cli": "17.4.4",
-    "@commitlint/config-conventional": "17.4.4",
-    "@commitlint/prompt-cli": "17.4.4",
-    "@lokalise/eslint-config-lokalise-frontend": "0.3.1",
-    "@semantic-release/changelog": "6.0.2",
-    "@semantic-release/commit-analyzer": "9.0.2",
-    "@semantic-release/git": "10.0.1",
-    "@semantic-release/github": "8.0.7",
-    "@semantic-release/npm": "9.0.2",
-    "@semantic-release/release-notes-generator": "10.0.3",
-    "eslint-config-prettier": "8.6.0",
-    "eslint-plugin-prettier": "4.2.1",
-    "husky": "8.0.3",
-    "prettier": "2.8.4",
-    "semantic-release": "20.1.0",
-    "typescript": "4.9.5",
-    "vite": "4.1.4",
-    "vite-plugin-dts": "2.0.2",
-    "vitest": "0.29.2"
-  }
+	"name": "@lokalise/your-project-name",
+	"version": "1.0.0",
+	"scripts": {
+		"build": "vite build",
+		"lint:eslint": "eslint --cache . --ext .js,.cjs,.ts",
+		"lint:ts": "tsc --noEmit",
+		"lint": "npm run lint:eslint && npm run lint:ts",
+		"lint:fix": "npm run lint:eslint -- --fix",
+		"test": "vitest run",
+		"prepare": "husky install",
+		"commit": "commit"
+	},
+	"type": "module",
+	"files": [
+		"dist"
+	],
+	"main": "./dist/your-project-name.umd.cjs",
+	"module": "./dist/your-project-name.js",
+	"exports": {
+		".": {
+			"import": "./dist/your-project-name.js",
+			"require": "./dist/your-project-name.umd.cjs"
+		}
+	},
+	"publishConfig": {
```

Comment:
Changed spaces to tabs.

The only other change is this `publishConfig` section which tells semantic-release to tell npm to make this project public when published.